src/style/content/conversation/message-list.less

```
@@ -87,7 +99,7 @@
 
 .content-message {
   &:has(.message-header) {
-    padding-top: 6px;
+    margint-top: 6px;
```

Typo spotted :nerd_face: 

```suggestion
    margin-top: 6px;
```